mobile/lib/modules/asset_viewer/views/video_viewer_page.dart

```diff
@@ -31,66 +25,17 @@ class VideoViewerPage extends HookConsumerWidget {
 
     String jwtToken = Hive.box(userInfoBox).get(accessTokenKey);
 
-    void showInfo() {
```

Comment:
Why do we remove these functions?

Reply:
GalleryView already does the same function.